packages/admin-rte/src/core/Controls/useBlockTypes.tsx

```diff
@@ -43,7 +44,14 @@ const createFeaturesFromBlocktypeMap = (group: "dropdown" | "button") => ({
         })),
 ];
 
-export default function useBlockTypes({ editorState, setEditorState, supportedThings, blocktypeMap, editorRef }: IProps) {
+export interface BlockTypes {
```

Comment:
I prefer to change the name to `BlockTypesApi`, the hook doesn't return "Blocktypes" per se, it returns a variety of "useful values" related to BlockTypes for other components to be used, so `...Api` seems reasonable to me.